README.md

```diff
@@ -4,27 +4,51 @@
+* [JSON Web Token authentication](#json-web-token-authentication)
+* [Handling callbacks](#handling-callbacks)
+* [Customising SDK](#customising-sdk)
+* [Uploading files to Onfido API](#uploading-files-to-onfido-api)
+* [Creating checks](#creating-checks)
+* [Going live](#going-live)
 
-This is a plug-and-play SDK that leverages the Onfido SDK core, helping users take document and face captures that can then be sent to our backend APIs.
+## Overview
 
-All document captures are sent over WebSockets to our image evaluation API, to ensure your users are submitting a document image of adequate quality.
+This SDK provides a set of [React](https://facebook.github.io/react/) components for JavaScript applications to allow capturing of identity documents and face photos for the purpose of identity verification. The SDK offers a number of benefits to help you create the best onboarding / identity verification experience for your customers:
 
-All document captures are collected through file upload. On handheld devices the SDK uses the `accept="image/*"` attribute to give the option to take a photo using the native capture methods. There is a feature currently in beta that uses the [getUserMedia API](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia) (where supported) to automatically detect and capture documents via a user’s webcam. This feature can be enabled using the [useWebcam](#public-options-and-methods) option. Face capture uses the webcam by default if one is available.
+- Carefully designed UI to guide your customers through the entire photo-capturing process
+- Modular design to help you seamlessly integrate the photo-capturing process into your application flow
+- Advanced image quality detection technology to ensure the quality of the captured images meets the requirement of the Onfido identity verification process, guaranteeing the best success rate
 
-To initialise the SDK, a connection to our WebSocket endpoint is required. Connections are authorized using [JWTs](https://jwt.io/), which can be generated on your server, or fetched from our JWT endpoint. Read about how to do this in the [authentication section](#authentication) below.
+Note: the SDK is only responsible for capturing photos. You still need to access the [Onfido API](https://documentation.onfido.com/) to upload photos, manage applicants and checks.
 
-## Screenshots
+All document captures are collected through browser file upload. On handheld devices the SDK uses the `accept="image/*"` attribute to give the option to take a photo using the native capture methods. There is a feature currently in beta that uses the [getUserMedia API](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia) (where supported) to automatically detect and capture documents via a user’s webcam. This feature can be enabled using the [useWebcam](#public-options-and-methods) option. Face capture uses the webcam by default if one is available.
```

Comment:
I remember that during the Integration Experience Design meeting we mentioned that this paragraph was a bit too technical to be part of the Overview. Might be worth moving it to #customising-sdk one?